resources/views/components/transaction/page/recipients.blade.php

```diff
@@ -0,0 +1,36 @@
+@props(['model'])
+
+@php
+    $recepients = $model->multiPaymentRecipients();
+@endphp
+
+<x-general.page-section.container
+    :title="trans('general.confirmations_only')"
```

Comment:
This should be "Recipients" - double check the langs to make sure it's still there